src/Valit/Rules/Extensions/ValitRuleDoubleExtensions.cs

```diff
@@ -4,7 +4,7 @@ namespace Valit
 {
     public static class ValitRuleDoubleExtensions
     {
-        public static IValitRule<TObject, double> IsGreaterThan<TObject>(this IValitRule<TObject, double> rule, double value) where TObject : class
+        public static IValitRule<TObject, double> IsGreaterThan<TObject>(this IValitRule<TObject, double> rule, double value, double epislon = 0.0d) where TObject : class
```

Comment:
Typo. Shouldn't be set to something more than 0?

Reply:
It is our default which means... ignore epsilon for now, unless you wanna set it explicitly for something like Double.Epsilon. ;)